### PR TITLE
Refactor/DEV-8375: Show warning in imageslice if file type belongs to warn list

### PR DESCRIPTION
**Pull request by hbalenda**
Changes:
- Show warning message when `imageslice` is passed an unsupported file type in the `warnList` of file extensions

@geealbers @Erin-Cecele Any suggested edits to the warning message? When `imageslice` attempts to process an unsupported file type in the `warnList` it will show the warning message, but continue to process other images.

### CODE REVIEW

File: packages/cli/lib/imageslice.js [hunk 1]
```diff
@@ -127,10 +127,11 @@ export default async function () {
           const dest = path.join(iiifProcessed, name);
 
           const supportedExts = [".jp2", ".jpg", ".jpeg", ".png", ".svg"];
+          const warnList = [".tiff", ".gif", ".bmp", ".webp", ".psd", ".heif", ".raw", ".pdf", ".ai", ".ind"];
```
Comment (mphstudios) [inline packages/cli/lib/imageslice.js:130]: It is always easier to read a sorted list...
```suggestion
          const warnList = [".tiff", ".gif", ".bmp", ".webp", ".psd", ".heif", ".raw", ".pdf", ".ai", ".ind"];
```

File: packages/cli/lib/imageslice.js [hunk 2]
```diff
@@ -127,10 +127,11 @@ export default async function () {
...
           if (supportedExts.some((supportedExt) => supportedExt === ext)) {
             originalImages.push(filePath);
-          } else {
-            spinner.fail(`Cannot slice file ${files[i]}. File type must be: ${supportedExts.join(', ')}.`);
+          } else if (warnList.some((warnExt) => warnExt === ext)) {
```
Comment (mphstudios) [inline packages/cli/lib/imageslice.js:133]:
```suggestion
          } else if (warnList.some((item) => item === ext)) {
```

File: packages/cli/lib/imageslice.js [hunk 3]
```diff
@@ -127,10 +127,12 @@ export default async function () {
           const dest = path.join(iiifProcessed, name);
 
           const supportedExts = [".jp2", ".jpg", ".jpeg", ".png", ".svg"];
+          // list of file extensions for common image types that can not be sliced into IIIF image tiles
+          const warnList = [".tiff", ".gif", ".bmp", ".webp", ".psd", ".heif", ".raw", ".pdf", ".ai", ".ind"];
```
Comment (hbalenda) [inline packages/cli/lib/imageslice.js:131]:
```suggestion
          const warnList = [".ai", ".bmp", ".gif", ".heif", ".ind", ".pdf", ".psd", ".raw", ".tiff", ".webp"];
```

**Review by mphstudios — APPROVED** [review]

**Comment by geealbers** [comment]
@hbalenda for the messaging, this is an example of what I see now:

```
galbers@GT29043 iiiflowers % quire process --iiif
ℹ Starting IIIF Process
ℹ Finding images to process...
✖ Cannot slice file .DS_Store. File type must be: .jp2, .jpg, .jpeg, .png, .svg.
✖ Cannot slice file fig-4-4.TIF. File type must be: .jp2, .jpg, .jpeg, .png, .svg.
✔ Done processing 3 images.
✖ 1/3 failed. Ensure these files have proper exif headings: 
 - fig-4-3.jpg
✔ 2/3 succeeded:
 - fig-4-13.jpg
 - fig-9-8.jpg
✔ IIIF Process Finished!
galbers@GT29043 iiiflowers % 
```

Does this PR override the messages like `Cannot slice file fig-4-4.TIF` or just adding on top of them, or ....?

**Comment by hbalenda** [comment]
@geealbers this PR only shows the `cannot slice file x` message if it is one of the file types in the "warning" list, which we can always update. You should no longer get a warning for files like `.DS_Store`, will you  make sure you're running the latest version on `main`?

I'll also be making a separate PR soon to fix the spinner and add a message about how the process might take a long time.

**Comment by geealbers** [comment]
@hbalenda sorry, that first time I wasn't running the latest version, I was just responding to your original query to Erin and I about the wording of the message. But, I just updated to the latest and ran it again, and now it's not giving me any `cannot slice` messages, including for the TIF that was still in the folder. 

```
galbers@GT29043 iiiflowers % quire process --iiif
ℹ Starting IIIF Process
ℹ Finding images to process...
✔ Done processing 3 images.
✖ 1/3 failed. Ensure these files have proper exif headings: 
 - fig-4-3.jpg
✔ 2/3 succeeded:
 - fig-4-13.jpg
 - fig-9-8.jpg
✔ IIIF Process Finished!
galbers@GT29043 iiiflowers %
```

**Comment by hbalenda** [comment]
@geealbers  ah, that is because they are all caps, I will update the list to normalize the case so we catch `.TIF` as well as `.tif`

**Comment by geealbers** [comment]
@hbalenda actually, I tried lowercase too and it still didn't work. though let me try again real quick to make sure. and maybe I'll through in a couple other files as well

**Comment by geealbers** [comment]
@hbalenda yeah, still not outputting any `cannot slice` messages at all

**Comment by hbalenda** [comment]
@geealbers `.tif` wasn't on the list, `.tiff` was. I will add `.tif` when I update the case handling.
